src/EnergyPlus/GroundHeatExchangers.hh

```diff
@@ -487,7 +487,7 @@ namespace GroundHeatExchangers {
     std::shared_ptr<GLHEResponseFactors>
```

Comment:
This `shared_ptr` stuff needs to be cleaned out. See `here` for a longer discussion, where I even used this module as an example! 📌